tensorflow/c/experimental/filesystem/filesystem_interface.h

```diff
@@ -880,22 +882,6 @@ typedef struct TF_FilesystemOps {
                                               TF_Filesystem_Option** option,
                                               TF_Status* status);
 
-  /// Sets the value of the filesystem option given in `key` to value in
-  /// `option`. Valid values of the `key` are returned by
-  /// `get_file_system_configuration_keys` call. Ownership of the `option` and
-  /// the `key` belogs to the caller. Buffers therein should be allocated and
-  /// freed by the filesystems allocation API.
-  ///
-  /// Plugins:
-  ///   * Must set `status` to `TF_OK` if `option` is set/updated
-  ///   * Must set `status` to `TF_NOT_FOUND` if the key is invalid
-  ///   * Might use any other error value for `status` to signal other errors.
-  ///
-  /// DEFAULT IMPLEMENTATION: return `TF_NOT_FOUND`.
-  void (*set_filesystem_configuration_option)(
-      const TF_Filesystem* filesystem, const TF_Filesystem_Option* option,
-      TF_Status* status);
```

Comment:
This should still exist as otherwise users will always need to get existing options, change the structure and then call the above method passing in all of the data. Plugins should allow setting just a single option.

Reply:
@mihaimaruseac Updated. That part has been restored.